app/services/query-manager.js

```diff
@@ -96,7 +96,7 @@ export default Ember.Service.extend({
 
   addResult(id, data) {
     return this.get('store').findRecord('query', id).then((query) => {
-      let result = this.get('store').createRecord('result', {
+      let result = Ember.run(this.get('store'), 'createRecord', 'result', {
```

Comment:
Why do we need another run loop again?

Reply:
I forget; let me see if it works without